pytest.ini

```diff
@@ -4,3 +4,7 @@ doctest_optionflags = ALLOW_UNICODE ELLIPSIS
 junit_suite_name = molecule_test_suite
 norecursedirs = dist doc build .tox .eggs
 testpaths = test/
+# filter out deprecation warning spam coming from pytest_verbose_parametrize,
+# since we can't do anything about it until the plugin is updated
```

Comment:
Make it actionable by adding a TODO/FIXME comment. Also, describe the effect of this change on the system and why someone in future would like to keep it and not delete thinking of refactoring.

Also, add a comment to https://github.com/ansible/molecule/blob/023bef688cc4e78301ec81cf3df42919f9817d08/tox.ini#L33 linking back here so that once we'll update that we'd backtrack the removal of this filtering as well.